cores/migrations/20240708182646_AddCategoryMarkdown.up.sql

```diff
@@ -1,13 +1,16 @@
--- Drop the existing primary key constraint
-ALTER TABLE Category DROP CONSTRAINT IF EXISTS category_pkey;
+-- Add category_id column ONLY if it doesn't exist
```

Comment:
I think this appraoch is safer because we don't alter the primary key to main the current data relationship.

I added a `catgory_id` column as a unique identifier

Reply:
This is PERFECT!

Love each step that you took here!

In a "big paranoid" company you might see teams split this into multiple migrations/deploys so they can watch each step. But I think this is great, more just chatting!